README.md

```diff
@@ -14,10 +14,10 @@ Then add the following line to your .bashrc file:
 
 That will create some directories inside your $HOME/.evm dir:
 
-- **erlang_tars** : the place where all erlang tarballs downloaded by evm will be cached.
+- **erlang_tars** : the place where all the erlang tarballs downloaded by evm will be cached.
```

Comment:
I noticed that myself, and I recognized the inconsistency, but I didn't change it before submitting.  I think you are correct, and it is better to be consistent